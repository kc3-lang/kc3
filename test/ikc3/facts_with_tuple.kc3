quote Facts.with_tuple(Facts.env_db(), {op = ?, :op_sym, sym = ?}, fn (fact) {
  puts(fact.object)
  void
})
Facts.with_tuple(Facts.env_db(), {op = ?, :op_sym, sym = ?}, fn (fact) {
  puts(fact.object)
  void
})
