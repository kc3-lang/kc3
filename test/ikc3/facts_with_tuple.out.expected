Facts.with_tuple(Facts.env_facts(), {KC3, :operator, ?}, fn (fact) { puts(fact.object); void })
operator_add
operator_addi
operator_and
operator_assign
operator_assign_2
operator_band
operator_bnot
operator_bor
operator_brackets
operator_bxor
operator_defstruct
operator_div
operator_eq
operator_equal
operator_gt
operator_gte
operator_lt
operator_lte
operator_mod
operator_mul
operator_neg
operator_not
operator_not_eq
operator_or
operator_paren
operator_pin
operator_require
operator_semicolumn
operator_shift_left
operator_shift_right
operator_sub
void
